content/03.methods.md

```diff
@@ -59,9 +61,9 @@ We provide examples for the categories, described below, using the following can
 
 **Databases**: These label functions incorporate existing databases to generate a signal, as seen in distant supervision [@doi:10.3115/1690219.1690287].
 These functions detect if a candidate sentence's co-mention pair is present in a given database.
-If the pair is present, emit a positive label and abstain otherwise.
-If the pair isn't present in any existing database, then a separate label function will emit a negative label.
-We use a separate label function to prevent the label imbalance problem. This problem occurs when candidates, that scarcely appear in databases, are drowned out by negative labels.
+If the candidate pair is present, our label function emitted a positive label and abstained otherwise.
+If the candidate pair wasn't present in any existing database, a separate label function emitted a negative label.
+We used a separate label function to prevent a label imbalance problem, which occurred when candidates that rarely appeared in databases were drowned out by negative labels.
```

Comment:
```suggestion
We used a separate label function to prevent a label imbalance problem that we encountered during development: emitting positive and negatives from the same label functions appeared to result in classifiers that predict almost exclusively negative predictions.
```